src/components/SuccessPage/SuccessPage.js

```diff
@@ -1,29 +1,29 @@
-import React from 'react';
+import React, {Component} from 'react';
 import './SuccessPage.css';
 import Logo from './../Logo/Logo';
 import Hand from './../Hand/Hand';
 import PropTypes from 'prop-types';
 
-const goHomeAndClearTimeout = timeOut =>  {
-        props.history.replace('/');
-        clearTimeout(timeOut);
-}
+class SuccessPage extends Component {
+  timeout = setTimeout(() => this.props.history.replace('/'), 10000);
```

Comment:
Want to set this in `componentDidMount` instead so that it resets after the first flow.